Requirements/ViewModels/RequirementBrowser/RequirementsBrowserViewModel.cs

```diff
@@ -153,21 +153,26 @@ public class RequirementsBrowserViewModel : ModellingThingBrowserViewModelBase,
                 pluginSettingsService)
         {
             this.Caption = $"{PanelCaption}, iteration_{this.Thing.IterationSetup.IterationNumber}";
+
             this.ToolTip =
                 $"{((EngineeringModel)this.Thing.Container).EngineeringModelSetup.Name}\n{this.Thing.IDalUri}\n{this.Session.ActivePerson.Name}";
 
             this.ReqSpecificationRows = new DisposableReactiveList<RequirementsSpecificationRowViewModel>();
             var model = (EngineeringModel)this.Thing.Container;
             this.ActiveParticipant = model.GetActiveParticipant(this.Session.ActivePerson);
+            this.openRequirementsSpecificationEditorViewModels = new List<RequirementsSpecificationEditorViewModel>();
 
-            this.ComputeUserDependentPermission();
-
-            this.UpdateRequirementSpecificationsRows();
+            this.ExecuteLongRunningDispatcherAction(
+                () => 
+                { 
+                    this.ComputeUserDependentPermission();
 
-            this.AddSubscriptions();
-            this.UpdateProperties();
+                    this.UpdateRequirementSpecificationsRows();
 
-            this.openRequirementsSpecificationEditorViewModels = new List<RequirementsSpecificationEditorViewModel>();
+                    this.AddSubscriptions();
+                    this.UpdateProperties();
+                },
+                "Loading browser...");
```

Comment:
i propose we make this specific to the browser, please rephrase to `Loading Requirements Browser`. I think the `PanelCaption` can be used here.